widget/embedded/src/hooks/useTheme.ts

```diff
@@ -30,9 +38,33 @@ export function useTheme({
       widget: fontFamily,
     },
   });
+  const darkColors = generateColors(mainColors, colors?.dark);
+  const lightColors = generateColors(mainColors, colors?.light);
+  const customLightTheme = Object.entries(lightColors).length
```

Comment:
Why are we using object.entries? can we use keys?

Reply:
fixed